docs/antctl.md

```diff
@@ -500,3 +508,117 @@ This feature is useful if one wants to use the Go
 [pprof](https://golang.org/pkg/net/http/pprof/) tool to collect runtime
 profiling data about the Antrea components. Please refer to this
 [document](troubleshooting.md#profiling-antrea-components) for more information.
+
+### Flow Aggregator commands
+
+Flow Aggregator supposts dumping flow records and printing record metrics.
```

Comment:
s/supposts/supports

Reply:
> antctl supports dumping the flow records handled by the Flow Aggregator, and printing metrics about flow record processing.

Reply:
Please emphasize here that these commands are only available when exec'ing into the Flow Aggregator Pod.

Reply:
done